configuration/parse.go

```diff
@@ -19,6 +22,9 @@ type Configuration struct {
 	Path    string
 }
 
+var unixComments = regexp.MustCompile(`#.*$`)
```

Comment:
Question: Should these be a const instead of a var?

Reply:
Nope golang doesn't allow structs to be constants. https://golang.org/ref/spec#Constants